docs/how-to-guides/access-monitoring.md

```diff
@@ -0,0 +1,81 @@
+# Access Monitoring on Lokomotive
+
+## Contents
+
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)
+* [Prometheus](#prometheus)
+* [Alertmanager](#alertmanager)
+* [Grafana](#grafana)
+  * [Using Port Forward](#using-port-forward)
+  * [Using Ingress](#using-ingress)
+
+## Introduction
+
+This guide explains how to access the monitoring stack. Lokomotive ships monitoring stack in the form of prometheus-operator component. Learn more about the prometheus-operator component and its parameters [here](../configuration-reference/components/prometheus-operator.md).
+
+## Prerequisites
+
+- Deploy the prometheus-operator component in `monitoring` namespace. If different namespace used in the configuration then change the subsequent commands accordingly, `kubectl -n <namespace> ...`.
+
+## Prometheus
+
+Execute the command to port forward prometheus locally on port `9090`:
+
+```
+kubectl -n monitoring port-forward svc/prometheus-operator-prometheus 9090:9090
+```
+
+Now open the following URL: [http://localhost:9090](http://localhost:9090).
+
+## Alertmanager
+
+Execute the command to port forward alertmanager locally on port `9093`.
+
+```
+kubectl -n monitoring port-forward svc/prometheus-operator-alertmanager 9093:9093
+```
+
+Now open the following URL: [http://localhost:9093](http://localhost:9093).
+
+## Grafana
+
+### Using Port Forward
+
+Execute the command to port forward grafana dashboard locally on port `8080`.
+
+```
+kubectl -n monitoring port-forward svc/prometheus-operator-grafana 8080:80
+```
+
+Obtain grafana `admin` user password by running the following command:
+
+```
+kubectl -n monitoring get secret prometheus-operator-grafana -o jsonpath='{.data.admin-password}' | base64 -d && echo
+```
+
+Now open the following URL: [http://localhost:8080](http://localhost:8080). Enter username `admin` and password obtained from the previous step.
+
+### Using Ingress
+
+To expose grafana to the internet using Ingress, provide the host field. The configuration for grafana in prometheus-operator should look like this:
```

Comment:
```suggestion
To expose grafana to the internet using Ingress, provide the host field. The configuration for grafana in prometheus-operator should look like the following:
```

"like this" is too informal IMO and doesn't fit the tone used in the rest of the guide.